tests/wcag/index.js

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env node
+
+const HttpServer = require('../test-utils/server');
+const { AxePuppeteer } = require('@axe-core/puppeteer');
+const StandardPageNavigator = require('../percy/standardpagenavigator');
+const WcagReporter = require('./wcagreporter');
+const puppeteer = require('puppeteer');
+const PORT = 5042;
+
+/**
+ * specifies options to be used by axe-core engine within axe-core/puppeteer.
+ * API documentation: https://github.com/dequelabs/axe-core/blob/master/doc/API.md
+ * - set reporter to 'no-passes' to only return violation results
+ * - set runOnly with tag values below to run WCAG standards:
+ *    - WCAG 2.0 Level A, AA, AAA
+ *    - WCAG 2.1 Level A, AA
+ */
+const config = {
+  reporter: "no-passes",
+  runOnly: {
+    type: 'tag',
+    values: ['wcag2a', 'wcag2aa', 'wcag21a', 'wcag21aa', 'wcag2aaa'],
+  }
+}
+
+async function wcagTester() {
+  const server = new HttpServer({
+    dir: 'test-site/public',
+    port: PORT
+  });
+  server.start();
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+  
+  const standardPageNavigator = new StandardPageNavigator(page, `http://localhost:${PORT}`);
```

Comment:
There are WCAG issues specific to the iFrame integration itself. For instance, with WCAG it's important that the iFrame have a title attribute, etc. For the purposes of this item (SLAP-1644) I think using the `StandardPageNavigator` is fine. But, I will make a followup to ensure we're testing WCAG compliance of the iFrame integration as well.

Reply:
I see, sounds good